app/views/pages/home.html.haml

```diff
@@ -1,7 +1,17 @@
 .jumbotron
-  %h1.display-4 Hello, world!
-  %p.lead This is a simple hero unit, a simple jumbotron-style component for calling extra attention to featured content or information.
-  %hr.my-4/
-  %p It uses utility classes for typography and spacing to space content out within the larger container.
+  %h1.display-4= t('.hello')
+  %p.lead= t('.create_service_book')
+  %p= t('.log_in_now')
   %p.lead
-    %a.btn.btn-primary.btn-lg{ href: "#", role: "button" } Learn more
+    %a.btn.btn-primary.btn-lg{ href: new_user_session_path, role: 'button' }
+      = t('.log_in')
+%h4.text-center= t('.why_is_worth')
+%h5= t('.we_have_several_reasons')
+.list-group
+  %a.list-group-item.list-group-item-action{ href: '#' }= t('.less_failures')
+  %a.list-group-item.list-group-item-action{ href: '#' }= t('.increase_value')
+  %a.list-group-item.list-group-item-action{ href: '#' }= t('.makes_you_happy')
+= image_tag 'home/motohelper.jpg', class: 'rounded mx-auto d-block'
```

Comment:
Great idea to add motohelper printscreen, but please add picture without OS information. Neutral solution. And maybe a little more information ;)

Make the image responsible.
https://getbootstrap.com/docs/4.0/content/images/#responsive-images